app/src/test/java/de/westnordost/streetcomplete/quests/shop_type/CheckShopExistenceTest.kt

```diff
@@ -5,14 +5,14 @@ import de.westnordost.streetcomplete.quests.TestMapDataWithGeometry
 import de.westnordost.streetcomplete.testutils.mock
 import de.westnordost.streetcomplete.testutils.node
 import de.westnordost.streetcomplete.testutils.on
-import java.util.concurrent.FutureTask
+import de.westnordost.streetcomplete.util.ktx.getFeature
 import kotlin.test.Test
 import kotlin.test.assertEquals
 
 class CheckShopExistenceTest {
-    private val questType = CheckShopExistence(mockOfFeatureDictionary())
+    private val questType = CheckShopExistence { mockOfFeatureDictionary().getFeature(it) }
```

Comment:
We don't need any mocking anymore in the tests. It's just a function now.

E.g. 

```
if (tags["shop"] == "greengrocer") return Feature(...)
else return null
```

Reply:
Ah, you're right. That actually simplifies mocking a lot!

Fixed in [`ea6264b`](https://github.com/streetcomplete/StreetComplete/pull/5273/commits/ea6264b06a2c758089a79ab121a398ab69088069).